src/main/java/com/google/sps/servlets/ConfirmationServlet.java

```diff
@@ -40,14 +41,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        // Get the id of the tutor whose availability will be displayed.
-        String tutorID = request.getParameter("tutorID");
+        // Get the id of the student whose availability will be displayed.
+        String studentEmail = request.getParameter("studentEmail");
 
         List<TutorSession> scheduledSessions = new ArrayList<TutorSession>();
 
-        for (Tutor tutor : SampleData.getSampleTutors()) {
-            if (tutorID.toLowerCase().equals(tutor.getEmail().toLowerCase())) {
-                scheduledSessions = Arrays.asList(tutor.getScheduledSessions());
+        for (Student student : SampleData.getSampleStudents()) {
+            if (studentEmail.toLowerCase().equals(student.getEmail().toLowerCase())) {
+                scheduledSessions = Arrays.asList(student.getScheduledSessions());
```

Comment:
Should there be a `break` in the if statement?

Reply:
That's true! Thanks for catching that